CHANGELOG

```diff
@@ -1,3 +1,13 @@
+# 1.0.10
```

Comment:
Just a minor comment for the future, you can include all of the below features into a single version bump.